ci-operator/step-registry/openstack/provision/vhostuser/openstack-provision-vhostuser-commands.sh

```diff
@@ -0,0 +1,158 @@
+#!/usr/bin/env bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+function check_workers_updating() {
```

Comment:
It's the third time I see the same function in this patch. I wish there was a way to share bash code between steps.

Reply:
the only way I see this is if we put this code in our own repo. I was thinking we could do this once we have the job in place. What do you think?